guides/common/modules/proc_configuring-automatic-removal-of-hosts-from-the-insights-inventory.adoc

```diff
@@ -0,0 +1,15 @@
+[id="configuring_automatic_removal_of_hosts_from_the_insights_inventory_{context}"]
+= Configuring Automatic Removal of Hosts from the Insights Inventory
+
+When hosts are removed from {Project}, they can also be removed from the inventory of Red{nbsp}Hat Insights, either automatically or manually.
+Use the following procedure to configure it automatically.
+
+.Prerequisite
+* Your user account must have a role assigned that grants the `view_settings` and `edit_settings` permissions.
```

Comment:
Working with SMEs and waiting for accurate values.

Reply:
The permissions that are needed to change the setting are the ones pointed out by Malhar: `view_settings` and `edit_settings`. 
There are two ways to change the setting: 
1. Through the settings page, where these two permissions should be enough
2. Through an API/CLI call, where these two permissions should be enough too
3. Through the Inventory upload page, where you have to have also the `view_foreman_rh_cloud` permission to view the page.

Reply:
Now it looks much better. Thanks Malhar!

Reply:
I'm not sure if the note is a good solution. The procedure we have below only describes the option to change the setting through the Inventory Upload page. If you want to document the alternative procedure, it should be done under the Procedure heading. I don't see a reason to document both, because then you would also have to document the recommendations sync in the same way. For your consideration, @mjivraja

Reply:
We only consider `Inventory Upload page` in this procedure so I agree with @Lennonka's view. If we are also considering `Settings page`, we will require a separate procedure for the same. 

IMO, we can stick with the former and original one to avoid confusion.

Reply:
I have to admit I am not even sure if you need the `edit_settings` permission in option 2.
According to the code, the permissions are not checked explicitly, which can be considered a bug.

Reply:
@ShimShtein @Lennonka Can we stick to confirmed permission `view_foreman_rh_cloud` as of now?

Reply:
Yes, sounds good.